Backend/DinoTracker/DinoTracker.graphql

```diff
@@ -8,7 +8,8 @@ type Query {
 }
 
 type Mutation {
-    createUser(name: String, username: String): Boolean!
+    createPaleontologist(name: String, username: String): Boolean!

```

Comment:
I like the name change. I've been pushing for us to start using Domain specific terms in code that the POs use. A user is more than just a user. They could be a paleontologist, a researcher, a guest... When people refer to users, usually they're just referring to a login that is linked to one of those. So it would make sense if our terminology reflected that